<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task4Window</class>
 <widget class="QDialog" name="Task4Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelQueue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Очередь</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стек</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerateStack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сгенерировать стек</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerateQueue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сгенерировать очередь</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditQueue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
